examples/simple_classifier_cifar10/start_with_hpo.py

```diff
@@ -4,7 +4,7 @@
 from syne_tune.backend.sagemaker_backend.sagemaker_utils import get_execution_role
 from syne_tune.config_space import choice, loguniform, uniform
 
-from renate.tuning import execute_tuning_job
+from renate.run import run_training_job
```

Comment:
what about `training` rather than `run`? We could also call the function simply `train` or `update`.

Reply:
I thought about update but since we have the `updaters` package, it can be quite confusing.
`training` can be OK, but pretty much everything we do is about training.

I don't mind changing the name but I don't see great alternatives

Reply:
We could also stick closer to the SM interface and create a `Renate` object on which we call `fit`. Since many arguments in SM and Renate are similar, might be a convenient interface for the experience SM user.

Reply:
Something like a `RenateEstimator`?

Reply:
In SM it is `sagemaker.pytorch.estimator.PyTorch` (no `Estimator`). That solution also has the advantage that we can potentially use the object to access specific properties of the job (model, hpo history etc)